### ISSUE TITLE
[Scheduler] Update out-of-date devel docs

### ISSUE THREAD

**Issue by kerthcet** [kubernetes/community#7300]
### Describe the issue

Some documentation under `contributors/devel/sig-scheduling` is out-of-date, we should update them. e.g. https://github.com/kubernetes/community/blob/master/contributors/devel/sig-scheduling/scheduler_queues.md

/sig scheduling

**Comment by Surajkumar98012** [comment]
hey i am new to open source but i want to contribute in this , can you guide me what to do

**Comment by kerthcet** [comment]
Do you familiar with kube scheduler, if not, this is a little hard for you because the doc is about the mechanism under the scheduler.

**Comment by Surajkumar98012** [comment]
yeah, I am familiar with Kube scheduler, I learnt about this when I am deep dive into the k8s architecture.

**Comment by kerthcet** [comment]
Basically, we have several changes to the scheduling queue:
- unschedulingQ -> unschedulingPods
- queueingHint introduced in the recent releases(beta but disabled by default)
- changed some metrics
- changed the background periodical flushing time

I think what we want to do here is just make the document updated with the latest changes.

**Comment by Surajkumar98012** [comment]
so, I think I need to compare it and update it with latest changes.
